### PR TITLE
Added maps support

### PR DESCRIPTION
**Pull request by noniagriconomie**
Re @marcaube 

Add map chart (cf https://github.com/marcaube/ObHighchartsBundle/issues/87)

map

### CODE REVIEW

**Comment by noniagriconomie** [comment]
@marcaube ping? is it possible to merge this asap? or need review?

**Comment by marcaube** [comment]
Sorry @noniagriconomie, I've been really busy with projects lately. I'll try to take a look at it (and other pending issues) tonight 👍

File: Resources/doc/cookbook.md [hunk 1]
```diff
@@ -162,3 +162,45 @@ $formatter = new Expr('function () {
...
+$map->series(array(
+    array(
+        'data' => array(
+            array('value' => rand(1, 100), 'code' => 'Île-de-France'),
+            array('value' => rand(1, 100), 'code' => 'Bourgogne'),
+            array('value' => rand(1, 100), 'code' => 'Alsace'),
+            // ...
+        ),
+        'name' => 'FR',
+        'mapData' => new \Zend\Json\Expr("Highcharts.maps['countries/fr/fr-all']"),
```
Comment (noniagriconomie) [inline Resources/doc/cookbook.md]: mapdata script, like the http://jsfiddle.net/gh/get/jquery/3.1.1/highslide-software/highcharts.com/tree/master/samples/maps/demo/color-axis/ example line 51

**Comment by noniagriconomie** [comment]
@marcaube hi,

does it need something more before merging it and go to packagist?

many thanks

**Pull request closed without merging**